archetypes/quickstart/src/brooklyn-sample/pom.xml

```diff
@@ -58,19 +53,30 @@
       <releases>
         <enabled>false</enabled>
       </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
     </repository>
   </repositories>
 
   <dependencies>
     <dependency>
-      <!-- this pulls in all brooklyn entities and clouds; 
-           you can cherry pick selected ones instead (for a smaller build) -->
+      <!-- minimal dependency, for writing entities; consider also brooklyn-software-base -->
```

Comment:
good questions @neykov which should be addressed on the web site, maybe the "code structure" section.  here i think we should link to web site, at most maybe include some commented out sample deps to include.